docs/changelog.rst

```diff
@@ -52,6 +52,7 @@ New features:
 
 Fixes:
 
+* :doc:`/beets/config_default`: Added default replace for - to _. :bug:`549`
```

Comment:
Aha; this is just a ReST syntax thing: the `:doc:` directive is for linking to other `*.rst` pages. We actually only really use this in the changelog for plugins, where the name of the ReST doc is the same as the name of the Python source file—here, you don't need to use a prefix. You might want to use ``:ref:`replace` `` inline in the sentence to link to the relevant config option in the configuration page, for example.